storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/MetricCollector.java

```diff
@@ -71,27 +81,17 @@ private Sensor createSensor(final String name) {
     public void publish(final MetricCollection metricCollection) {
         final List<String> metricValues = metricCollection.metricValues(CoreMetric.OPERATION_NAME);
         for (final String metricValue : metricValues) {
-            // Try to arrange them by likelihood, at least GetObjectRequest should be first.
```

Comment:
I don't think it really matters in this case since these are request metrics and the request itself is a lot slower anyways, same reason for using `Sensor#record`